slack/context.w

```diff
@@ -0,0 +1,145 @@
+bring "./message.w" as msg;
+bring "./util.w" as helpers;
+bring "./events.w" as events;
+
+/// The bahvioral interface of a thread
+pub inflight interface IThread {
+  inflight postMessage(message: msg.Message): Json;
```

Comment:
```suggestion
  postMessage(message: msg.Message): Json;
```

Since the entire interface is inflight I believe it shouldn't be needed to specify this for each method. No?